golang/cosmos/x/swingset/types/msgs.go

```diff
@@ -72,6 +72,16 @@ func (msg MsgDeliverInbound) GetInboundMsgCount() int32 {
 	return 1
 }
 
+// IsHighPriority implements the vm.ControllerAdmissionMsg interface.
+func (msg MsgDeliverInbound) IsHighPriority(ctx sdk.Context, data interface{}) (bool, error) {
+	_, ok := data.(SwingSetKeeper)
```

Comment:
Consider just returning `false, nil` regardless of the `data` argument. If not, consider commenting on why not. Here and below.

Reply:
Yeah I was wondering about that. I did this for future maintainability, but I wasn't sure of the runtime cost.

Reply:
The runtime cost is inconsequential - I was thinking of the maintainability impact of the clutter. Even in an ideal world where the argument was compile-time type checked, you probably wouldn't do further checks to make sure it was a _valid_ keeper value - you'd just ignore it and immediately return false. For the same reason, I think it's fine to just ignore the untyped value when you don't use it.

Reply:
> you probably wouldn't do further checks to make sure it was a _valid_ keeper value - you'd just ignore it and immediately return false

In the case where it's used for `MsgWalletSpendAction`, I do need to cast it to a keeper of the right interface to use it first. I basically kept that structure from that check and replicated it to the other message types.

However I agree we can always copy back from a functioning method if we need to add support to other message types.